module/x/peggy/types/key.go

```diff
@@ -38,7 +38,7 @@ var (
 	KeyLastOutgoingBatchID = append(SequenceKeyPrefix, []byte("lastBatchId")...)
 )
 
-func GetEthAddressKey(validator sdk.AccAddress) []byte {
+func GetEthAddressKey(validator sdk.ValAddress) []byte {
```

Comment:
what exactly is the difference between a valAddress and an AccAddress?

Reply:
Different use cases:
* `ValAddress` is the validator's operator address
* `AccAddress` is an address that can hold tokens

For us the orchestrator will use an `AccAddress` and `EthereumAddress` that we need to link to the `ValAddress` so that we can identify to which validator the orchestrator belongs. For key rotation or other reasons an orchestrator may want to change his key/ address with Peggy. We would then just update the link. The `ValAddress` won't change during the whole process.